app/tables/nuntius/messages_table.rb

```diff
@@ -8,6 +8,16 @@ class MessagesTable < Nuntius::ApplicationTable
       column(:to)
       column(:created_at)
       column(:last_sent_at)
+      column(:actions) do
+        render do
+          html do |message|
+            link_to(nuntius.resend_admin_message_path(message), title: 'Resend Message', data: { turbo_method: :post }, class: 'bg-gray-200 text-black p-1 w-6 items-center flex rounded dark:bg-gray-700 dark:text-white') do
+              content_tag(:i, nil, class: 'fal fa-rotate-right')
+            end
+          end
+        end
+      end
```

Comment:
Here on sorting arrow getting error, need to remove sorting for these kind of columns.

```
PG::UndefinedColumn: ERROR:  column "actions" does not exist
LINE 1: ...s_messages.metadata->>'account_id' = $1) ORDER BY "actions" ...
```
Thank you.

Reply:
Mensa has a separate actions syntax, see readme.
Don't add a column for the mensa ones :)